.circleci/config.yml

```diff
@@ -63,6 +68,9 @@ jobs:
       - run:
           name: Check dApp Licenses
           command: cd ~/src/raiden-dapp && license-checker --production --onlyAllow 'MIT;BSD;Apache;MPL;LGPL;ZPL;ISC;WTFPL;Unlicense' --excludePackages 'uuid@2.0.1' --excludePrivatePackages # uuid@2.0.1 has an issue with the license information but it is licensed under MIT
+      - run:
+          name: Check CLI Licenses
+          command: cd ~/src/raiden-cli && license-checker --production --onlyAllow 'MIT;BSD;Apache;MPL;LGPL;ZPL;ISC;WTFPL;Unlicense' --excludePackages 'uuid@2.0.1' --excludePrivatePackages # uuid@2.0.1 has an issue with the license information but it is licensed under MIT
```

Comment:
Nitpick:
We have basically the same line 3 times with the same comment copied. Since this is a mono repo we can't know if there isn't more to come. But even these three are enough in my opinion to make this somewhat re-usable. If you agree and like the idea I would recommand to create a [custom command](https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands) and add the directory as a parameter for this command. This approach might be nice for some other places as well. :shrug:

Reply:
Sure let's open an issue for that :)

Reply:
Issue: #1739